src/hooks/useFeatureFlag.js

```diff
@@ -1,16 +1,29 @@
+import { FeatureFlags } from '@/constants/FeatureFlags';
+import Cookies from 'universal-cookie';
 import { useCookiesWithOptions } from './useCookiesWithOptions';
 
+const cookies = new Cookies();
+
+const createFullIdentifier = (identifier) => `ff_${identifier}`;
+
 const useFeatureFlag = (identifier) => {
   if (!identifier) return [false, () => {}];
 
-  const fullIdentifier = `ff_${identifier}`;
-
   const { cookies, setCookie } = useCookiesWithOptions();
 
+  const fullIdentifier = createFullIdentifier(identifier);
+
   const set = (value) => setCookie(fullIdentifier, value);
   const value = cookies?.[fullIdentifier] ?? false;
 
   return [value, set];
 };
 
+if (typeof window !== 'undefined') {
+  window.FeatureFlags = FeatureFlags;
+  window.setFeatureFlag = (identifier, value) => {
+    cookies.set(createFullIdentifier(identifier), value);
+  };
+}
```

Comment:
No comment but just a question: I'm curious why we need these on the `window` object? To use them manually in the `console` or something?

Reply:
Nevermind I saw your explanation on Slack now 👍

Reply:
idd, exposing a function on window makes them 'global' to use in the console